docs/02-deployment-guide.md

```diff
@@ -161,8 +176,9 @@ directory.
 The Sleeper CLI runs commands inside a Docker container. This way you can avoid needing to install any of the
 dependencies or build Sleeper yourself.
 
-The `sleeper deployment` command gets you a shell inside a Docker container as though you were in the scripts directory
-of the Sleeper Git repository. The rest of the repository will not be present.
+The `sleeper builder` command gets you a shell inside a Docker container with the Sleeper Git repository checked out to
+the `sleeper` directory. This docker container will have all the dependencies required to build and deploy an
```

Comment:
This isn't quite true. The Git repository is only checked out when you create a `sleeper environment` deployment, or add a user to the environment EC2. If you run this locally you only get the dependencies, though you can check out the repository manually.